pkg/codegen/pcl/binder_test.go

```diff
@@ -163,29 +162,13 @@ func TestWritingProgramSource(t *testing.T) {
 	assert.True(t, exampleMainExists, "main program file of example component should exist")
 }
 
-func parseAndBindProgram(t *testing.T, text, name string, options ...pcl.BindOption) (*pcl.Program, hcl.Diagnostics) {
-	parser := syntax.NewParser()
-	err := parser.ParseFile(strings.NewReader(text), name)
-	if err != nil {
-		t.Fatalf("could not read %v: %v", name, err)
-	}
-	if parser.Diagnostics.HasErrors() {
-		t.Fatalf("failed to parse files: %v", parser.Diagnostics)
-	}
-
-	options = append(options, pcl.PluginHost(utils.NewHost(testdataPath)))
-
-	program, diags, err := pcl.BindProgram(parser.Files, options...)
-	if err != nil {
-		t.Fatalf("could not bind program: %v", err)
-	}
-	return program, diags
-}
-
 func TestConfigNodeTypedString(t *testing.T) {
 	t.Parallel()
 	source := "config cidrBlock string { }"
-	program, diags := parseAndBindProgram(t, source, "config.pp")
+	program, diags, err := ParseAndBindProgram(t, source, "config.pp")
+	if err != nil {
```

Comment:
can use "github.com/stretchr/testify/require" and require.NoError(t, err) for these. Saves a load of `if err != nil` lines.

Reply:
Will refactor shortly

Reply:
Done ✅